templates/service-template.yml

```diff
@@ -1085,14 +1080,9 @@ objects:
             - --observatorium-timeout=${OBSERVATORIUM_TIMEOUT}
             - --observatorium-auth-type=${OBSERVATORIUM_AUTH_TYPE}
             - --observatorium-token-file=/secrets/service/observatorium.token
```

Comment:
Wouldn't the set of changes in this PR require the addition of new parameters and flag setting in the service template?